data/mjr2595.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Michael Raymond",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Just a guy trying to dev stuff",
+  "avatar": "https://github.com/mjr2595.png",
+  "links": [
+    {
+      "name": "Git over here",
+      "url": "https://github.com/mjr2595",
+      "icon": "FaGithubAlt"
+    },
+    {
+      "name": "Link with me",
+      "url": "https://www.linkedin.com/in/mjraymond",
+      "icon": "FaLinkedinIn"
```

Comment:
Wrong icon name.
```suggestion
      "icon": "FaLinkedin"
```

Reply:
Hi @Panquesito7! 

I'm seeing "FaLinkedinIn" as an available icon when searching https://linkfree.eddiehub.io/icons

![image](https://user-images.githubusercontent.com/7285657/214982636-6f4b7f52-6be5-4047-833c-432bb754caae.png)

I'm ok with making this change if this alt icon isn't really available. Although I would also then post an issue 😄.